lib/pengine/clone.c

```diff
@@ -279,6 +280,30 @@ short_print(char *list, const char *prefix, const char *type, const char *suffix
     }
 }
 
+static void
+short_output(char *list, const char *type, long options, pcmk__output_t * out)
+{
+    char buffer[LINE_MAX];
+
+    if (list) {
+        if (options & pe_print_html) {
```

Comment:
We are checking the options flags all over the place and is not generic. We decided to leave the options flags for now

> The options argument is a little trickier. The new functions should definitely ignore the format-related values (pe_print_log/html/ncurses/printf/xml). For now, we can keep the options argument for the remaining values, but we're planning the ability to specify options in the output object, so it may eventually fit into that.

So maybe, we could use `out->begin_list(out, "li", NULL, NULL);` to make a list item tag for now? Or, should I call a message like `out->message(out, "list-item", ...)` ?

Reply:
Whenever we have a function that checks the format, that's an indication that it should be a custom message. Just as an example, not necessarily what's needed here:

    { "short-output", "xml", short_output_xml },
    { "short-output", "html", short_output_html },
    { "short-output", "text", short_output_text },

and each short_output_*() function would have the part relevant to that format.